README.md

```diff
@@ -1,6 +1,11 @@
-# flutter-clean-architecture
+## Introduction
```

Comment:
Include `h1` "**Flutter Clean Scope**" before "**Introduction**".

Reply:
Resolved

Reply:
`h1` in Markdown is used with `#`, it should be `# Flutter Clean Scope`.

Reply:
Resolved `# Flutter Clean Scope`